apps/homepage/src/app/layout.tsx

```diff
@@ -16,7 +18,12 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={`${sen.variable}`}>{children}</body>
+      <body className={`${firaCode.className}`}>
+        <ViewWrapper>
```

Comment:
Looks like the theme is not changing to dark on the home page app: 

<img width="1726" alt="Screenshot 2024-12-17 at 7 15 20 PM" src="https://github.com/user-attachments/assets/fa477d37-2134-4ec2-a90f-d2426371f7f2" />